dev/sparktestsupport/modules.py

```diff
@@ -611,43 +611,47 @@ def __hash__(self):
         "pyspark.pandas.spark.utils",
         "pyspark.pandas.typedef.typehints",
         # unittests
-        "pyspark.pandas.tests.test_dataframe",
+        "pyspark.pandas.tests.indexes.test_base",
```

Comment:
Looks like we could add some testcase discover mechanism on it in follow up.

Reply:
Sounds good! That would reduce maintenance costs.

Reply:
Finally, it is merged in https://github.com/apache/spark/pull/32867 , : )

Reply:
Thanks for making it happen! :)